Integrations/integration-Aella_StarLight.yml

```diff
@@ -0,0 +1,331 @@
+description: Aella Star Light Integration
+configuration:
+- display: Server URL (e.g. https://starlight.companyname.com:8889)
+  name: url
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: User name
+  name: credentials
+  defaultvalue: ""
+  type: 9
+  required: true
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
+- display: Fetching interval in minutes (default is 15, minimum is 15 )
+  name: fetch_interval
+  defaultvalue: "15"
+  type: 0
+  required: false
+- display: The specific security event to look for. Default is all events
+  name: event_name
+  defaultvalue: ""
+  type: 0
+  required: false
+- display: Security event severity threshold, between 0-100
+  name: severity
+  defaultvalue: "50"
+  type: 0
+  required: false
+- display: Trust any certificate (unsecure)
+  name: insecure
+  defaultvalue: "true"
+  type: 8
+  required: false
+script:
+  script: |-
+    import requests
+    import json
+    import time
+    import datetime
+
+    import urllib3
+    urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+    from requests.packages.urllib3.exceptions import InsecureRequestWarning
+    requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
+
+    ''' GLOBAL VARS '''
+    URL = demisto.getParam('url') + '/aellaelastic'
```

Comment:
Not sure what do you mean here.
The Display for the parameter has this:
Server URL (e.g. https://starlight.companyname.com:8889)

It will also work for https://starlight.companyname.com:8889/

Can you give me an example what is the exact input error?

Reply:
demoed that is not a problem. Leave it as is.